bench/irmin-pack/trace_definitions.ml

```diff
@@ -0,0 +1,118 @@
+(** Traces file format definitions.
+
+    This file is also meant to be used from Tezos. OCaml version 4.09 and the
+    32bit architecture should be supported.
+
+    {3 Traces Workflow}
+
+    A Tezos node (may) output a [Replayable_trace] file (TODO: Or a Raw traces,
```

Comment:
a replayable trace is a raw trace + postprocessing? maybe reformulate to say that